// react-native-instagram-story-circle/build/dist/IGStoryCircle.style.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports._notificationContainer = exports._profileImageStyle = exports._storyRing = void 0;
const react_native_1 = require("react-native");
exports._storyRing = (size = 64, borderRadius) => ({
    width: size,
    height: size,
    borderRadius: borderRadius ? borderRadius : size / 2,
    alignItems: "center",
    justifyContent: "center",
});
exports._profileImageStyle = (innerCircleSize, profileImageBorderSize, borderRingColor, borderRadius) => ({
    width: innerCircleSize,
    height: innerCircleSize,
    borderColor: borderRingColor,
    borderWidth: profileImageBorderSize,
    borderRadius: borderRadius ? borderRadius : innerCircleSize,
});
exports._notificationContainer = (notificationSize, notificationBubbleBackgroundColor, notificationPositionTop, notificationPositionLeft) => ({
    position: "absolute",
    alignItems: "center",
    justifyContent: "center",
    width: notificationSize,
    height: notificationSize,
    top: notificationPositionTop,
    left: notificationPositionLeft,
    borderRadius: notificationSize,
    backgroundColor: notificationBubbleBackgroundColor,
});
exports.default = react_native_1.StyleSheet.create({
    container: {
        alignItems: "center",
        justifyContent: "center",
    },
    insertIcon: {
        width: 8,
        height: 8,
    },
    notificationText: {
        fontSize: 10,
        color: "#fdfdfd",
        fontWeight: "bold",
    },
});
